src/fontinfo.rs

```diff
@@ -5,15 +5,178 @@ use serde::{Deserialize, Serialize};
 use crate::shared_types::Guideline;
 use crate::Error;
 
-// The specification is vague about data type limits, usually implicitly meaning
-// Python types. Since Python is dynamic, the spec does not nail down the exact type
-// in several locations and we have to assume a bigger type that can hold all sane
-// values.
-type Integer = i32;
-type NonNegativeInteger = u32;
-type IntegerOrFloat = f64;
-type Float = f64;
-type Bitlist = Vec<u8>;
+/// Fontinfo value types are put into a module so that code on this level cannot access the
+/// literal constructors.
+mod types {
```

Comment:
I might just put this in a separate file? That should give us the same guarantees.

Reply:
Moved it into shared_types.

Maybe I need to re-export `*IntegerOrFloat`?